Gemfile

```diff
@@ -1,12 +1,13 @@
-ruby "2.2.4"
+ruby RUBY_VERSION
```

Comment:
@timrogers Hi again, well, it's a not-often-used "hint feature" of Bundler that can be used by gem authors. I documented it here, https://github.com/bundler/bundler/pull/5200 - but with luck the next release of Bundler will do this version navigation unassisted.